test/server.test.js

```diff
@@ -1,14 +1,471 @@
 const chai = require('chai');
 const chaiHttp = require('chai-http');
 const server = require('../app/server'); // Adjust the path as needed
+const { serve } = require('swagger-ui-express');
+const { sendChaiGet, sendChaiGetWithHeader, sendChaiPost, sendChaiPostWithHeader, sendChaiDelete } = require("../utility/chaiHelpers");
+const { randId, testProduct} = require("../utility/helper");
 
 const should = chai.should();
 chai.use(chaiHttp);
 
-// TODO: Write tests for the server
+describe('Brands', () => {
+  
+  describe("/GET brands/", () => {
+    it("should GET all brands", (done) => {
+      sendChaiGet(server, "/api/brands", done, (err, res, done) => {
+        res.should.have.status(200)
+        res.body.should.be.an("array")
```

Comment:
This is not really testing anything, if you change your actual code to return an empty array, this test wont fail and you will be introducing a severe production bug. Always test agains mocks to make sure you are returning the proper data.